libs/cart/README.md

```diff
@@ -3,9 +3,30 @@
 Building and maintaining a model and code for an ecommerce store is complex and mentally taxing. `@daffodil/cart`
 provides clear interfaces, models, and factories for the frontend of an ecommerce store so that you don't have to.
 
-
 ## Installation
 
-```
-npm install @daffodil/cart
-```
+Refer to [the installation guide](./guides/installation).
```

Comment:
These links don't seem to work. https://github.com/griest024/daffodil/blob/docs/cart/guides/libs/cart/README.md

Reply:
hm, they worked in the VSCode markdown preview. I might need to add the file extension

Reply:
added file extensions, hopefully they work now?